Global install writes resolved absolute paths that break in Docker containers

## Bug

When GSD is installed on Windows, `bin/install.js` resolves `os.homedir()` into the `pathPrefix` used for all `.md` file path replacements. This produces paths like `C:/Users/matte/.claude/get-shit-done/bin/gsd-tools.cjs` in ~43 workflow/reference/agent files.

When `~/.claude` is then mounted into a Docker container (e.g. a devcontainer), Node.js interprets the Windows absolute path as **relative to CWD**, producing:

```
/workspace/project/C:/Users/matte/.claude/get-shit-done/bin/gsd-tools.cjs
```

This causes `MODULE_NOT_FOUND` errors for any GSD command that invokes `gsd-tools.cjs`.

## Reproduction

1. Install GSD globally on Windows (`node bin/install.js -g`)
2. Mount `~/.claude` into a Docker container (Linux)
3. Run any GSD command that invokes `gsd-tools.cjs` (e.g. `/gsd:plan-phase`)
4. Observe: `Cannot find module '/workspace/project/C:/Users/matte/.claude/get-shit-done/bin/gsd-tools.cjs'`

## Root cause

In `bin/install.js`, the `pathPrefix` computation at line ~2416 always uses resolved absolute paths:

```js
const pathPrefix = `${path.resolve(targetDir).replace(/\\/g, '/')}/`;
```

On Windows this produces `C:/Users/matte/.claude/` which is baked into all installed `.md` files.

## Proposed fix

For global installs, replace `os.homedir()` with `~` in the path prefix so files contain `~/.claude/get-shit-done/...` instead of resolved absolute paths. Local installs can keep absolute paths since they may be outside `$HOME`.

```js
const pathPrefix = isGlobal
  ? path.resolve(targetDir).replace(os.homedir(), '~').replace(/\\/g, '/') + '/'
  : `${path.resolve(targetDir).replace(/\\/g, '/')}/`;
```

## Related issues

- #926 — relative @-paths break when CWD != $HOME
- #786 — subagents rewrite absolute gsd-tools.cjs paths → MODULE_NOT_FOUND
- #143 — $HOME doesn't expand on Windows in hooks

--- a/bin/install.js
+++ b/bin/install.js
@@ -2411,9 +2411,12 @@ function install(isGlobal, runtime = 'claude') {
 
   // Path prefix for file references in markdown content (e.g. gsd-tools.cjs).
   // Replaces $HOME/.claude/ or ~/.claude/ so the result is <pathPrefix>get-shit-done/bin/...
-  // Always use absolute path so: (1) local installs work when GSD is outside $HOME,
-  // (2) spawned subagents with empty $HOME still resolve the path (fixes #820).
-  const pathPrefix = `${path.resolve(targetDir).replace(/\\/g, '/')}/`;
+  // For global installs: use ~/ so paths work across environments (e.g. Docker
+  // containers mounting ~/.claude from a Windows host where os.homedir() differs).
+  // For local installs: use resolved absolute path (may be outside $HOME).
+  const pathPrefix = isGlobal
+    ? path.resolve(targetDir).replace(os.homedir(), '~').replace(/\\/g, '/') + '/'
+    : `${path.resolve(targetDir).replace(/\\/g, '/')}/`;
 
   let runtimeLabel = 'Claude Code';
   if (isOpencode) runtimeLabel = 'OpenCode';

--- a/tests/path-replacement.test.cjs
+++ b/tests/path-replacement.test.cjs
@@ -1,0 +1,100 @@
+/**
+ * GSD Tests - path replacement in install.js
+ *
+ * Verifies that global installs produce ~/ paths in .md files,
+ * never resolved absolute paths containing os.homedir().
+ * Reproduces the bug where Windows installs write C:/Users/...
+ * paths that break in Docker containers.
+ */
+
+const { test, describe } = require('node:test');
+const assert = require('node:assert');
+const fs = require('fs');
+const path = require('path');
+const os = require('os');
+
+const repoRoot = path.join(__dirname, '..');
+
+// Simulate the pathPrefix computation from install.js (global install)
+function computePathPrefix(homedir, targetDir) {
+  return path.resolve(targetDir).replace(homedir, '~').replace(/\\/g, '/') + '/';
+}
+
+describe('pathPrefix computation', () => {
+  test('default Claude global install uses ~/', () => {
+    const homedir = os.homedir();
+    const targetDir = path.join(homedir, '.claude');
+    const prefix = computePathPrefix(homedir, targetDir);
+    assert.strictEqual(prefix, '~/.claude/');
+  });
+
+  test('default Gemini global install uses ~/', () => {
+    const homedir = os.homedir();
+    const targetDir = path.join(homedir, '.gemini');
+    const prefix = computePathPrefix(homedir, targetDir);
+    assert.strictEqual(prefix, '~/.gemini/');
+  });
+
+  test('custom config dir under home uses ~/', () => {
+    const homedir = os.homedir();
+    const targetDir = path.join(homedir, '.config', 'claude');
+    const prefix = computePathPrefix(homedir, targetDir);
+    assert.ok(prefix.startsWith('~/'), `Expected ~/ prefix, got: ${prefix}`);
+    assert.ok(!prefix.includes(homedir), `Should not contain homedir: ${homedir}`);
+  });
+
+  test('Windows-style paths produce ~/ not C:/', () => {
+    // On Windows, path.resolve returns the input unchanged when it's already absolute.
+    // Simulate the string operation directly (can't use path.resolve for Windows paths on Linux).
+    const winHomedir = 'C:\\Users\\matte';
+    const winTargetDir = 'C:\\Users\\matte\\.claude';
+    // This is what the fix does: targetDir.replace(homedir, '~').replace(/\\/g, '/') + '/'
+    const prefix = winTargetDir.replace(winHomedir, '~').replace(/\\/g, '/') + '/';
+    assert.strictEqual(prefix, '~/.claude/');
+    assert.ok(!prefix.includes('C:'), `Should not contain drive letter, got: ${prefix}`);
+  });
+});
+
+describe('installed .md files contain no resolved absolute paths', () => {
+  const homedir = os.homedir();
+  const targetDir = path.join(homedir, '.claude');
+  const pathPrefix = computePathPrefix(homedir, targetDir);
+  const claudeDirRegex = /~\/\.claude\//g;
+  const claudeHomeRegex = /\$HOME\/\.claude\//g;
+  const normalizedHomedir = homedir.replace(/\\/g, '/');
+
+  // Collect all .md files from source directories
+  function collectMdFiles(dir) {
+    const results = [];
+    if (!fs.existsSync(dir)) return results;
+    for (const entry of fs.readdirSync(dir, { withFileTypes: true })) {
+      const fullPath = path.join(dir, entry.name);
+      if (entry.isDirectory()) {
+        results.push(...collectMdFiles(fullPath));
+      } else if (entry.name.endsWith('.md')) {
+        results.push(fullPath);
+      }
+    }
+    return results;
+  }
+
+  const dirsToCheck = ['commands', 'get-shit-done', 'agents'].map(d => path.join(repoRoot, d));
+  const mdFiles = dirsToCheck.flatMap(collectMdFiles);
+
+  test('source .md files exist', () => {
+    assert.ok(mdFiles.length > 0, `Expected .md files, found ${mdFiles.length}`);
+  });
+
+  test('after replacement, no .md file contains os.homedir()', () => {
+    const failures = [];
+    for (const file of mdFiles) {
+      let content = fs.readFileSync(file, 'utf8');
+      content = content.replace(claudeDirRegex, pathPrefix);
+      content = content.replace(claudeHomeRegex, pathPrefix);
+      if (content.includes(normalizedHomedir) && normalizedHomedir !== '~') {
+        failures.push(path.relative(repoRoot, file));
+      }
+    }
+    assert.deepStrictEqual(failures, [], `Files with resolved absolute paths: ${failures.join(', ')}`);
+  });
+});

